src/main/java/it/gov/pagopa/pu/bff/dto/ReceiptViewFiltersDTO.java

```diff
@@ -0,0 +1,23 @@
+package it.gov.pagopa.pu.bff.dto;
+
+import it.gov.pagopa.pu.debtpositions.dto.generated.ReceiptView;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.OffsetDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ReceiptViewFiltersDTO {
+  private Long organizationId;
+  private ReceiptView.ReceiptOriginEnum receiptOrigin;
+  private String operatorExternalUserId;
+  private String iuv;
+  private String iur;
+  private String iud;
+  private Long debtPositionTypeOrgId;
+  private OffsetDateTime fromDate;
+  private OffsetDateTime toDate;
```

Comment:
let's introduce a common DTO to express OffsetDateTimeInterval like done in:
https://github.com/pagopa/p4pa-process-executions/blob/develop/src/main/java/it/gov/pagopa/pu/processecexutions/dto/LocalDateIntervalFilter.java
```suggestion
  private OffsetDateTimeIntervalFilter paymentDateTime;
```